docs/ui-components/callouts.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Callouts
+parent: UI Components
+nav_order: 7
+---
+
+# Callouts
+
+Markdown does not include support for callouts. However, you can style text as a callout using a Markdown extension supported by kramdown: [*block IALs*](https://kramdown.gettalong.org/quickref.html#block-attributes). 
+
+Common kinds of callouts include `admonition`, `attention`, `caution`, `danger`, `error`, `hint`, `important`, `note`, `tip`, and `warning`.
+
+When you have [configured]({{ site.baseurl }}{% link docs/configuration.md %}#callouts) the  `color` and (optional) `title` for a callout, you can apply it to a paragraph, or to a block quote with several paragraphs, as illustrated below.[^postfix]
+
+[^postfix]:
+    You can put the callout markup either before or after its content.
+
+#### An untitled callout
```

Comment:
This breaks semantic HTML heading order. Do you think `##`/`<h2>` would be appropriate to use here?

Reply:
Similar breaks occur in the existing docs pages – e.g., see the following near the top of [navigation-structure.md](https://github.com/pdmosses/just-the-docs/blob/combination/docs/navigation-structure.md):

```markdown
## Ordering pages

To specify a page order, you can use the `nav_order` parameter in your pages' YAML front matter.

#### Example
```

I followed the same pattern for the sake of visual uniformity with the (unobtrusive) heading style used to introduce examples.

In general, however, I do prefer to keep to semantic HTML. A compromise here would be to use the same *visual* style as `<h4>` (and add an anchor to facilitate citation in discussions of examples). But ideally, that should be done uniformly for all docs pages…

I suggest that we leave the `####`s in [callouts.md](https://github.com/pdmosses/just-the-docs/blob/combination/docs/ui-components/callouts.md), and revisit this docs-only issue after v0.4.0.

Reply:
Sounds good - I'll write up an issue for this.

Reply:
Have just made #870!